text/0000-roadmap-2018.md

```diff
@@ -0,0 +1,412 @@
+>
+> [Releasing “Rust 2018” gives us a chance to say to the world that “Rust has taken a major step since 1.0; it’s time to take another look”.](http://aturon.github.io/blog/2018/01/09/rust-2018/) (@aturon)
+
+The Rust epoch 2018 release encompasses every aspect of the work we do, so we’ll look at each area in turn. **This RFC is not intended as a** ***promise*** **about what will ship, but rather a strong (and realistic) intention.** The core team will ultimately oversee the precise timing and feature set of the release.
+
+It’s important to keep in mind two additional factors:
+
+
+- “Shipping” features in this context means they must be *stable*. We may land additional unstable features this year (like `const` generics), but these are separate from the Rust 2018 product. **We will stabilize features individually as they become ready,** ***not*** **in a rush before the epoch release.**
+
+
+- The intent is to ship Rust 2018 in the latter part of the year. The tentative target date is the 1.29 release, which goes into beta on 2018-08-02 and ships on 2018-09-13. That gives us approximately six months to put the product together.
+
+**These two factors together suggest that Rust 2018 will ship largely with** ***language*** **features that are already in nightly in some form today.** Other, faster-moving areas of the product will be developing new material throughout the year.
+
+As always, we will continue to push out new Rust releases on a six week cadence, so a given feature missing the epoch release is by no means fatal. On the other hand, we need to carefully coordinate the work so that the features we *do* ship sit together coherently across the compiler, tools, documentation, libraries and marketing materials.
+
+#### Language
+
+
+> [Rust 2018: Consolidation (](https://deterministic.space/rust-2018.html)@killercup)
+
+The most prominent language work in the pipeline stems from [2017’s ergonomics initiative](https://blog.rust-lang.org/2017/03/02/lang-ergonomics.html). Almost all of the accepted RFCs from the initiative are available on nightly, but polish, testing, and consensus work will take time:
+
+[](https://boats.gitlab.io/blog/post/2018-01-08-goals-for-rust/)
+> [I’d like to reach a final decision to ship or drop all of the ergonomics RFCs that were accepted. I hope to see this completed over the next several mont](https://boats.gitlab.io/blog/post/2018-01-08-goals-for-rust/)[h](https://boats.gitlab.io/blog/post/2018-01-08-goals-for-rust/)[s . . . I hope we can have a clearer (and more spaced out) schedule for this so that their FCPs are staggered.](https://boats.gitlab.io/blog/post/2018-01-08-goals-for-rust/) (@withoutboats)
+
+Among these productivity features are a few “headliners” that will form the backbone of the release:
+
+
+- **Non-lexical lifetimes**. Currently in “alpha” state on nightly, with work ongoing.
+- `**impl Trait**`. Nearing readiness for stabilization FCP.
```

Comment:
Very minor nit, but this is misformatted; I think for the markdown to render correctly, the backticks should go _inside_ the asterisks:
```
**`impl Trait`**
```
rather than 
```
`**impl Trait**`
```